packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -380,10 +387,10 @@ export class Outbox {
 				0x5a6 /* Compression should not have happened if the loader does not support it */,
 			);
 
-			this.params.legacySendBatchFn(batch);
+			clientSequenceNumber = this.params.legacySendBatchFn(batch);
```

Comment:
Do we know for sure this returns clientSequenceNumber?

Reply:
Yes - it was like this before #13994 

And for fun I browsed back via git blame... 2019 was the last time I can't immediately prove it returns a CSN.
https://github.com/microsoft/FluidFramework/pull/839
https://github.com/microsoft/FluidFramework/commit/05a5df0106238896ac2913d2bae356ccd3cde4e3